EDSAnalytics/EDSTest/README.md

```diff
@@ -0,0 +1,105 @@
+# EDS Analytics Sample and Test
```

Comment:
This sample needs a HISTORY.md file, see [HISTORY.md](https://github.com/osisoft/OSI-Samples-OCS/blob/master/basic_samples/Authentication/AuthorizationCodeFlow/DotNet/HISTORY.md) for an example. Right now should just include 1.0.0 / {date}, Initial public release

Reply:
Done.